lib/src/widget/bottom_sheet/bottom_sheet_body_mixin.dart

```diff
@@ -1,25 +1,11 @@
 import 'package:flutter/material.dart';
 import 'package:modal_bottom_sheet/modal_bottom_sheet.dart';
-import 'package:xayn_design/src/widget/nav_bar/widget/nav_bar_container.dart';
 
 mixin BottomSheetBodyMixin {
   ScrollController? getScrollController(BuildContext context) =>
       ModalScrollController.of(context);
 
```

Comment:
do you still need the mixin?

Reply:
I want to keep it for two reasons:
1. in case some bottom sheets are draggable in the future, we can use the scroll controller here.
2. in case we want to add or change logic when closing the bottom sheet, it will be abstracted away, instead of using "pop". A PR by Carmine, will introduce "onClose" to the bottomSheet for example.

Reply:
@joecks

Reply:
mäh ... 
its fine. Just were thinking about this  in general, its seems a bit arbitary to have the dependency on xayn_design because it doesn't really solve any design issue that is very particular for xayn. Anyway. nothing blocking here really.